ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -1,4 +1,16 @@
 <PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs">
+      <ul>
+        <li>
+          <span class="sep">&#x0002f;</span>
+          <LinkTo @route={{concat "vault.cluster.access.oidc.scopes" (unless @model.isNew ".scope.details")}}>
```

Comment:
So this means that it will navigate back to the details view if we're in editing an existing scope? Nice job! I like this (and can confirm with design)

Reply:
Yes back to details when on the edit view and back to the list view when on create.